src/index.js

```diff
@@ -1,79 +1,2 @@
-import { filter, find, forEach } from 'min-dash';
-
-import extractors from './extractors';
-
-import { selfAndAllFlowElements, getParents, getElement } from './util/ElementsUtil';
-
-/**
- * @typedef {Object} ProcessVariable
- * @property {string} name
- * @property {Array<ModdleElement>} origin
- * @property {ModdleElement} scope
- */
-
-// api /////////////////////////
-
-/**
- * Retrieves all process variables for a given container element.
- * @param {ModdleElement} containerElement
- *
- * @returns {Array<ProcessVariable>}
- */
-export function getProcessVariables(containerElement) {
-  var processVariables = [];
-
-  // (1) extract all flow elements inside the container
-  var elements = selfAndAllFlowElements([containerElement], false);
-
-  // (2) extract all variables from the extractors
-  forEach(extractors, function(extractor) {
-    extractor({
-      elements: elements,
-      containerElement: containerElement,
-      processVariables: processVariables
-    });
-  });
-
-  return processVariables;
-}
-
-/**
- * Retrieves all variables which are available in the given scope
- *
- * * Exclude variables which are only available in other scopes
- * * Exclude variables which are produced by the given element
- * * Include variables which are available in parent scopes
- *
- * @param {string} scope
- * @param {ModdleElement} rootElement element from where to extract all variables
- *
- * @returns {Array<ProcessVariable>}
- */
-export function getVariablesForScope(scope, rootElement) {
-
-  var allVariables = getProcessVariables(rootElement);
-
-  var scopeElement = getElement(scope, rootElement);
-
-  // (1) get variables for given scope
-  var scopeVariables = filter(allVariables, function(variable) {
-    return variable.scope.id === scopeElement.id;
-  });
-
-  // (2) get variables for parent scopes
-  var parents = getParents(scopeElement);
-
-  var parentsScopeVariables = filter(allVariables, function(variable) {
-    return find(parents, function(parent) {
-      return parent.id === variable.scope.id;
-    });
-  });
-
-  return combineArrays(scopeVariables, parentsScopeVariables);
-}
-
-// helpers ////////////////////
-
-function combineArrays(a, b) {
-  return a.concat(b);
-}
+export * from './platform';
```

Comment:
Shouldn't we simply export the Zeebe related functionality under a different prefix (i.e. `extract-process-variables/zeebe`)? 

I personally like that more, especially given that cases where we use both are likely rare.

Reply:
Great Idea, let's to that instead! This way, we don't need the clunky `extractZeebeVariables` and can keep the same name scheme

Reply:
I believe it's the way to go, too.

Reply:
Implemented in [088547e](https://github.com/bpmn-io/extract-process-variables/pull/20/commits/088547efa4c54035773c874e131170636793e8ca)